docs/docs/overview/quickstart/get-started-with-kotlin.md

```diff
@@ -0,0 +1,288 @@
+
+## Create a Subscriber and Update a Subscriber's information
+
+The recipients of a triggered notification are called subscribers.
+
+Click **Subscribers** on the left sidebar of the Novu dashboard to see all subscribers. By default, the dashboard will display a subscriber, as you were added automatically during sign-up.
+
+![subscribers.png](https://res.cloudinary.com/dxc6bnman/image/upload/v1688331839/Screenshot_2023-07-03_at_0.02.53_jmkhi3.png)
+
+Now, let's create a subscriber on Novu. Copy and paste the following code to do so:
+```kotlin
+suspend fun createSubscriber(): ResponseWrapper<SubscriberResponse>? {
+        val novu = Novu(apiKey = "API_KEY")
+        val subscriberRequest = SubscriberRequest(
+            email = "email@sample.com",
+            firstName = "John",//optional
+            lastName = "Doe",//optional
+            phone = "123456789",//optional
+            avatar = "sample-avatar",//optional
+            subscriberId = "12345"//optional
+        )
+        return CoroutineScope(Dispatchers.IO).async {
+            novu.createSubscriber(subscriberRequest)
+        }.await()
+    }
+```
+When you run this code snippet, you should see the subscriber on your Novu dashboard. :bust_in_silhouette:
+
+Now, let's assume you want to modify the email address of the Subscriber you just created, you can do that with the following code:
+```kotlin
+suspend fun updateSubscriber(): ResponseWrapper<SubscriberResponse>? {
+        val novu = Novu(apiKey = "API_KEY")
+        val subscriberRequest = UpdateSubscriberRequest(
+            email = "validemail@sample.com",
+            firstName = "John",//optional
+            lastName = "Doe",//optional
+            phone = "123456789",//optional
+            avatar = "sample-avatar"//optional
+        )
+        val subscriberId = "12345"
+        return CoroutineScope(Dispatchers.IO).async {
+            novu.updateSubscriber(subscriberId, subscriberRequest)
+        }.await()
+    }
+```
+Other valid fields that can be updated are `phone`, `avatar`, and `data` . The `data` field can accept an array of metadata that you want to attach to the subscriber.
+
+::::info
+To create all of your subscribers, you need to programmatically add them to Novu.
+::::
+
+## Trigger a Notification
+
+Copy and paste the following code into your app to trigger a notification to a Subscriber:
+```kotlin
+suspend fun triggerNotification(): Any? {
+        val novu = Novu(apiKey = "API_KEY")
+        val triggerEventRequest = TriggerEventRequest(
+            name = "test",
+            to = SubscriberRequest(
+                subscriberId = "12345",
+                email = "email@email.com",
+                firstName = "John",
+                lastName = "Doe"
+            ),
+            payload = mapOf("customVariables" to "Hello"),
+            transactionId = "transactionId"
```

Comment:
I think we can leave out the transactionId. It's optional and here we are just assigning a random string to it. Let's remove it.